packages/fast-jss-manager-react/src/manage-jss.tsx

```diff
@@ -8,7 +8,7 @@ import jss, { stylesheetManager } from "./jss";
 import { SheetsManager, StyleSheet } from "jss";
 import { IDesignSystemProviderProps } from "./design-system-provider";
 import * as propTypes from "prop-types";
-import { ClassNames, ComponentStyles, IManagedClasses } from "@microsoft/fast-jss-manager";
+import { ClassNames, ComponentStyles, ComponentStyleSheet, IManagedClasses } from "@microsoft/fast-jss-manager";
 import { isEqual, merge, omit } from "lodash-es";
 
 // hoist-non-react-statics does not seem to be a properly formatted ES6 module, so we need to require it instead
```

Comment:
Is this a known bug?  Seems like something we should tag with todo and issue number so we can clean up at later time when the dependency is fixed.

Reply:
sure